src/common/snippets/tests/src/lowered/pass/split_loops.cpp

```diff
@@ -407,14 +408,24 @@ TEST_F(SplitLoopsTest, BrgemmAddBrgemm) {
         const auto& loop_manager = linear_ir_ref->get_loop_manager();
 
         // Brgemm1 n-blocking loop
-        const auto brgemm1_n_loop = std::make_shared<UnifiedLoopInfo>(
+        const auto blocking_n_loop = std::make_shared<UnifiedLoopInfo>(
```

Comment:
We finally expect blocking_n_loop should fuse k block of second brgemm(first brgemm with loop on m and n. Second brgemm with loop on m and k). Does that should be done in separate PR?

Reply:
I believe this should be supported within a separate task since it may require some major changes in common logic. The current feature already unblocks OnlineSoftmax support from SplitLoops logic perspective, and affects the existing real cases, so In my opinion, the current state of the PR is sufficient for merging